docs/transactions/payment-request/create_payment_request.md

```diff
@@ -0,0 +1,52 @@
+---
+description: Create a payment request b58 code to give to someone else
+---
+
+# Create Payment Request
+
+## Parameters
+
+| Required Param | Purpose | Requirements |
+| :--- | :--- | :--- |
+| `account_id` | The account on which to perform this action. | Account must exist in the wallet. |
+| `amount_pmob` | The amount of pMOB to send in this transaction. | `u64` |
+
+| Optional Param | Purpose | Requirements |
+| :--- | :--- | :--- |
+| `subaddress_index` | The subaddress index on the account to send the transaction | `i64` |
+| `memo` | Memo for the payment request |  |
+
+## Example
+
+{% tabs %}
+{% tab title="Request Body" %}
+```text
+{
+  "method": "create_payment_request",
+  "params": {
+    "account_id": "a8c9c7acb96cf4ad9154eec9384c09f2c75a340b441924847fe5f60a41805bde",
+    "amount_pmob": 42000000000000,
+    "subaddress_index": 4,
+    "memo": "Happy Birthday!"
```

Comment:
Odd nit, but this memo might be confusing. You're asking someone to pay you and saying them happy birthday?